pkg/apis/agones/v1/gameserver.go

```diff
@@ -82,6 +82,8 @@ const (
 	// Passthrough dynamically sets the `containerPort` to the same value as the dynamically selected hostPort.
 	// This will mean that users will need to lookup what port has been opened through the server side SDK.
 	Passthrough PortPolicy = "Passthrough"
+	// None means the `hostPort` is ignored and if defined, the `containerPort` (optional) is used to set the port on the GameServer instance.
+	None PortPolicy = "None"
```

Comment:
Since we're in here, shall we also update:

https://github.com/googleforgames/agones/blob/981f79abce919cdb5496354cef934892f63bf1f0/pkg/apis/agones/v1/gameserver.go#L277-L281 ?

Will also need to run `make gen-api-docs gen-crd-client` to ensure the comment changes pass all the way through.